source/codegen/metadata/nifgen/CHANGES.md

```diff
@@ -0,0 +1,56 @@
+# Changes in metadata from the nimi-python metadata
+
+## config.py
+
+The following metadata was added:
+- 'c_function_prefix': 'niFgen_'
+- 'service_class_prefix': 'NiFgen'
+- 'java_package': 'com.ni.grpc.fgen'
+- 'csharp_namespace': 'NationalInstruments.Grpc.Fgen'
+
+## attributes.py
+
+The documentation and lv_property values were removed from all the attributes.
+
+`enum` tag was removed from the following attributes:
```

Comment:
Something seems wrong here. Things you have listed here are not attributes.

Reply:
He has mentioned the list of values he has removed from enums, these will be added back in the subsequent PRs. For now he has removed because they were giving build errors. @vijayinvites maybe you can just mention the enum names here instead of listing all the values, or you could actually not mention anything here in CHANGES.md since we will be adding them back anyway.

Reply:
Removed those changes from the changes.md file.